### ISSUE TITLE
unauthorized device 

### ISSUE THREAD

**Issue by randomdudeee** [Genymobile/gnirehtet#192]
```
C:\Users\ShatterDome\Desktop\gnirehtet>gnirehtet install
2019-05-26 14:21:36.768 INFO Main: Installing gnirehtet client...
* daemon not running; starting now at tcp:5037
* daemon started successfully
error: device unauthorized.
This adb server's $ADB_VENDOR_KEYS is not set
Try 'adb kill-server' if that seems wrong.
Otherwise check for a confirmation dialog on your device.
error: device unauthorized.
This adb server's $ADB_VENDOR_KEYS is not set
Try 'adb kill-server' if that seems wrong.
Otherwise check for a confirmation dialog on your device.
Performing Streamed Install
adb: connect error for write: device unauthorized.
This adb server's $ADB_VENDOR_KEYS is not set
Try 'adb kill-server' if that seems wrong.
Otherwise check for a confirmation dialog on your device.
2019-05-26 14:21:41.400 ERROR Main: Execution error: Command adb ["install", "-r", "gnirehtet.apk"] returned with value 1
```
plz help me with this error

**Comment by rom1v** [comment]
You need to authorize your computer on the device. Do you have a confirmation dialog?

Check:

    adb devices

(it's related to _adb_, independant of _gnirehtet_)

**Comment by randomdudeee** [comment]
how do i authorize. normally when i install gnirehtet the authorize screen pops up. but this device don't show any pop ups

**Comment by rom1v** [comment]
> how do i authorize. normally when i install gnirehtet the authorize screen pops up. but this device don't show any pop ups

This is a problem different from "unauthorized device" on _adb_.

IIUC, you don't get this popup:
![popup](https://raw.githubusercontent.com/Genymobile/gnirehtet/2bf8a3cfbe2168031897b8f917c088fe6d47710c/assets/request.jpg)

Unfortunately, this is a recurrent problem. Some devices require to "authorize" the app manually, but each in different ways. See for example: https://github.com/Genymobile/gnirehtet/issues/161#issuecomment-455324982 and https://github.com/Genymobile/gnirehtet/issues/150#issuecomment-440989256.

**Comment by rom1v** [comment]
Should be fixed by f8681809a28a0073f791f0d9eae65da02ae73c3d.
